src/data/nat/factorization.lean

```diff
@@ -199,10 +199,14 @@ begin
   { rintro ⟨c, rfl⟩, rw factorization_mul hd (right_ne_zero_of_mul hn), simp },
 end
 
-lemma prime_pow_dvd_iff_le_factorization (p k n : ℕ) (pp : prime p) (hn : n ≠ 0) :
+lemma prime.pow_dvd_iff_le_factorization {p k n : ℕ} (pp : prime p) (hn : n ≠ 0) :
   p ^ k ∣ n ↔ k ≤ n.factorization p :=
 by rw [←factorization_le_iff_dvd (pow_pos pp.pos k).ne' hn, pp.factorization_pow, single_le_iff]
 
+lemma prime.pow_dvd_multiplicity_iff {p k n : ℕ} (pp : prime p) (hn : n ≠ 0) :
```

Comment:
One last comment: The lemma name mentions `multiplicity` rather than `factorization`. Would something like `prime.pow_dvd_iff_dvd_pow_factorization` be better?

Reply:
Hmm, it doesn't really divide the factorization as such, but I can't come up with a better name, so maybe simply list the symbols appearing in the statement as you suggest.

Reply:
(and I think if you want to allow me to merge you have to delegate, not just click the approval button)

Reply:
Can I delegate a delegation?

bors d+

Reply:
Oh, I guess I can!